### DISCUSSION TITLE
Reinitialize to a different python environment

### DISCUSSION THREAD

**Discussion by DanWBR** [pythonnet/pythonnet#1875] [Q&A] [answered]
Hi,

Is it possible to initialize PythonEngine to a given python environment, then shut it down and initialize it again to another environment in a different path?

Thanks

**Comment by lostmsu** [comment, marked as answer]
@DanWBR that is not supported. You need to start a new process.

**Reply by DanWBR** [reply]
Thanks 👍🏻
